docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_subscription.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: PROCEDURE
+:troubleshooting_builds:
+
+[id="subscription_{context}"]
+= Using subscription
```

Comment:
Maybe "Using Red Hat subscriptions" or "Using RPM subscriptions" would be more obvious? "Subscription" by itself could mean anything :D